pkg/mdformatter/linktransformer/config.go

```diff
@@ -52,12 +55,22 @@ type IgnoreValidator struct {
 	_regex *regexp.Regexp
 }
 
+type LocalValidator struct {
+	_regex *regexp.Regexp
+}
+
+type IgnoreLocalValidator struct {
+	_regex *regexp.Regexp
```

Comment:
Let's use exactly the same validator routines for local links, so we don't need to add any extra config bit.

Probably to ensure compatibility with current use cases, we might want to add special flag for it `--enable-explicit-local-validators`

Reply:
Keeping it as a YAML config option instead, as we mention validators via that too. 🙂